plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -108,22 +116,23 @@
   delegate_to: localhost
 
 - name: Create GitLab Project in group Ansible
-  gitlab_project:
+  community.general.gitlab_project:
     api_url: https://gitlab.example.com/
     validate_certs: True
     api_username: dj-wasabi
     api_password: "MySecretPassword"
     name: my_first_project
     group: ansible
     issues_enabled: False
+    merge_method: rebase_merge
     wiki_enabled: True
     snippets_enabled: True
     import_url: http://git.example.com/example/lab.git
     state: present
   delegate_to: localhost
 '''
 
-RETURN = '''
+RETURN = r'''
 msg:
   description: Success or failure message
```

Comment:
```suggestion
  description: Success or failure message.
```
according to https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block